docker/standalone/cpu/Dockerfile

```diff
@@ -0,0 +1,155 @@
+FROM ubuntu:14.04
+MAINTAINER arun.das@my.utsa.edu
+
+RUN apt-get update && apt-get install -y software-properties-common
+RUN add-apt-repository ppa:openjdk-r/ppa
```

Comment:
This line is repeated in 37 as well. Sorry about that. Didn't see it while uploading. Makes no errors, but redundant.